src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide2CurrentMagnitudeEquationTerm.java

```diff
@@ -58,23 +58,26 @@ public void update(double[] x) {
         double sinPh2 = FastMath.sin(ph2);
         double cosPh2Ksi = FastMath.cos(ph2 + ksi);
         double sinPh2Ksi = FastMath.sin(ph2 + ksi);
-        double theta = ksi - (a1Var != null && a1Var.isActive() ? x[a1Var.getRow()] : branch.getPiModel().getA1())
-            + A2 + ph1;
+        double theta = ksi + (a1Var != null && a1Var.isActive() ? x[a1Var.getRow()] : branch.getPiModel().getA1())
+                - A2 + ph1;
```

Comment:
Maybe we should have more something like:

double theta = ksi + A1
                -  (a2Var != null && a2Var.isActive() ? x[a2Var.getRow()] : branch.getPiModel().getA2()) + ph1;